content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,473 @@
+# How to build a task manager with angular.
+
+### Introduction
+As developers, it is common practice to build simple applications. This will help solidify one's knowledge about a particular programming language or framework. 
+
+In this article, we will build a simple task manager application (AKA To-do app) using [Angular](https://angular.io/docs). Angular is a JavaScript framework.
+
+### Table of content
+-  Prerequisites
+-  Introduction
+-  Key takeaways
+-  What is a Task Manager
+-  Create a new angular project
+-  Create a component
+-  Add task
+-  Mark task as done
+-  Delete a task
+-  Save to local storage
+-  Summary
+
+
+### Prerequisites
+Before we get started, here are a few prerequisites for developing a task manager via Angular;
+
+- You should have [Node.js](https://nodejs.org/en/) installed on your computer.
+- You should be fairly familiar with [Angular](https://angular.io/docs) and JavaScript. Knowledge of [TypeScript](https://www.typescriptlang.org/) is useful but not requisite.
+
+### Key takeaways
+At the end of this tutorial, you should be able to:
+-  create and structure an angular project.
+-  create a component in Angular.
+-  build a simple application.
+-  save items to local storage
+
+
+### What is a Task Manager?
+A task manager writes, organizes, and rearranges tasks more efficiently. The task manager we will build will allow the user to add a task, mark the task as complete, and delete the task.
+
+
+### Create a new angular project
+Before creating a new angular project, one will need to install angular CLI. Run `npm install -g @angular/cli` on the terminal or command prompt to install the CLI.
+
+   ```npm install -g @angular/cli
+   ```
```

Comment:
This is a command. You should have something like:
 ```bash 
some text goes here 
```
This can be achieved by adding ```bash then closing it with the same. Check markdown docs.